src/public/modules/forms/base/controllers/submit-form.client.controller.js

```diff
@@ -33,17 +33,11 @@ function SubmitFormController(
 
   // React migration checker - ONLY for plain form URLs (no suffixes like /template or /preview)
   if (/^\/[0-9a-fA-F]{24}\/?$/.test($location.path())) {
-    const adminCookie = $cookies.get($window.reactMigrationAdminCookieName)
     const respondentCookie = $cookies.get(
       $window.reactMigrationRespondentCookieName,
     )
 
-    if (adminCookie) {
-      if (adminCookie === 'react') {
-        $window.location.href = `/${form._id}`
-        return
-      }
-    } else if (respondentCookie === 'react') {
+    if (respondentCookie === 'react') {
       $window.location.href = `/${form._id}`
```

Comment:
/nit, Is it possible to use the `location.assign()` functional API in this case?